web/Web/web/static/js/document-viewer.js

```diff
@@ -556,6 +563,28 @@ docView.prototype = {
       });
     }
 
+      function getScalInfo(callback) {
+        const url = options.getScalInfoURL;
+        $.ajax({
+          url: url,
+          method: 'GET',
+          success: function (result) {
+            if (result) {
+              let temp = [result['stratum_number'], result['stratum_size'], result['sample_size'], (parent.viewStack.length + 1).toString()]
+              Array.from(document.getElementById('scal-info').getElementsByTagName('small')).forEach((span, i) => {
+                span.innerHTML = temp[i]
+              })
+            }
+            if (typeof callback === "function") {
+              callback();
+            }
+          },
+          error: function (err_msg) {
+            showError(JSON.stringify(err_msg));
```

Comment:
Calling `showError`  will change the whole document view and we don't want that if there is an error with fetching the information.

I suppose we can still call the new function that will update the html, but instead of updating with actual numbers, it will just update them with `NA` or something.